resources/views/layouts/layout.blade.php

```diff
@@ -55,6 +59,145 @@
                     @endif
                 </ul>
             </li>
+            <li class="nav-item">
+                <a class="nav-link text-dark {{ (request()->segment(1) == 'faq') ? 'font-weight-bold' : '' }}" href="/vragenantwoorden">FAQ</a>
+            </li>
+            <li class="nav-item">
+                <a class="nav-link text-dark {{ (request()->segment(1) == 'nieuwsbrief') ? 'font-weight-bold' : '' }}" href="/nieuwsbrief">Nieuws</a>
+            </li>
+            <li class="nav-item">
+                <a class="nav-link text-dark {{ (request()->segment(1) == 'team') ? 'font-weight-bold' : '' }}" href="/team">Team</a>
+            </li>
+            <li class="nav-item">
+                <a class="nav-link text-dark {{ (request()->segment(1) == 'partner') ? 'font-weight-bold' : '' }}" href="/partner">Partners</a>
+            </li>
+            <li class="nav-item">
+                <a class="nav-link text-dark {{ (request()->segment(1) == 'overons') ? 'font-weight-bold' : '' }}" href="/overons">Over Ons</a>
+            </li>
+            <li class="nav-item">
+                <a class="nav-link text-dark {{ (request()->segment(1) == 'locatie') ? 'font-weight-bold' : '' }}" href="/locatie">Locatie</a>
+            </li>
+            <li class="nav-item">
+                <a class="nav-link text-dark {{ (request()->segment(1) == 'links') ? 'font-weight-bold' : '' }}" href="/links">Links</a>
+            </li>
+            <li class="nav-item">
+                <a class="nav-link text-dark {{ (request()->segment(1) == 'login') ? 'font-weight-bold' : '' }}" href="/login">Login</a>
```

Comment:
Ik denk zelf dat de inlogknop mooier/gebruikelijker is volledig aan de rechterkant

Reply:
fixed